modules/tinymce/changelog.txt

```diff
@@ -11,6 +11,7 @@ Version 5.7.0 (TBD)
     Changed the z-index of the `setProgressState(true)` throbber so it does not hide notifications #TINY-6686
     Changed the type signature for `editor.selection.getRng()` incorrectly returning `null` #TINY-6843
     Changed some `SaxParser` regular expressions to improve performance #TINY-6823
+    Fixed `col` elements were incorrectly transformed to `th` when converting columns to header columns #TINY-6715
```

Comment:
```suggestion
    Fixed `col` elements incorrectly transformed to `th` elements when converting columns to header columns #TINY-6715
```
This just read weirdly, as it was like `Fixed` was used as a header, not part of the sentence.